indigo_app/templates/indigo_api/_task_cards.html

```diff
@@ -91,9 +91,20 @@ <h6 class="text-center mb-4">
                 {% endfor %}
               </div>
 
+              {% if task.last_assigned_to and task.state == 'pending_review' %}
+                <div>
+                  <i class="fas fa-user-tag fa-xs task-icon-assignee" title="Submitted by"></i>
+                  <small class="text-muted">{% user_profile task.last_assigned_to %}</small>
+                </div>
+              {% endif %}
+
               {% if task.assigned_to %}
                 <div>
-                  <i class="fas fa-user-tag fa-xs task-icon-assignee" title="Assigned to"></i>
+                  {% if task.state == 'pending_review' %}
+                    <i class="fas fa-user-check fa-xs task-icon-assignee" title="Assigned to"></i>
```

Comment:
I think the icon for "assigned to" shouldn't change. It should be different from the "approved by" icon, though, which I think Alicia used `fa-user-check` for.

Reply:
fair enough
we'll need a new icon for 'submitted by' then – maybe user-edit? 
![image](https://user-images.githubusercontent.com/32566441/58413868-22d10580-807a-11e9-8a4e-d9dd51463872.png)